docs/releaseNotes.yml

```diff
@@ -32,6 +32,19 @@
 
 changelog: https://github.com/emissary-ingress/emissary/blob/$branch$/CHANGELOG.md
 items:
+  - version: 3.2.0
+    date: '2022-08-15'
```

Comment:
Could not find this documented. I tried to set a placeholder, but it complained because it didn't know how to parse the date. TBD does the trick, thanks.